Cargo.toml

```diff
@@ -33,6 +33,7 @@ ruma-state-res = { version = "0.11.0", path = "crates/ruma-state-res" }
 serde = { version = "1.0.164", features = ["derive"] }
 serde_html_form = "0.2.0"
 serde_json = "1.0.87"
+serde_with = "3.8.1"
```

Comment:
I would very much like to avoid this dependency. It's pretty heavy in terms of compile time.

Reply:
Okay; I was mostly just trying it out to see if their implementation worked when mine didn't.